diopter/sanitizer.py

```diff
@@ -129,6 +129,7 @@ class Sanitizer:
         "excess elements in struct initializer",
         "comparison between pointer and integer",
         "return type of ‘main’ is not ‘int’",
+        "return type of 'main' is not 'int'",
```

Comment:
Irrelevant change

Reply:
No, it's on purpose, the tests fail without this, the different quote styles where appearing in tests